packages/@groupby/elements-cache-driver-plugin/src/cache-driver-plugin.ts

```diff
@@ -0,0 +1,63 @@
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved
+import { Plugin, PluginMetadata, PluginRegistry } from '@groupby/elements-core';
+import { CACHE_REQUEST, CacheRequestPayload, CacheResponsePayload } from '@groupby/elements-events';
+
+/**
+ * The GroupBy Elements cache driver plugin.
+ * This plugin exposes an event-based interface to the GB Elements cache.
+ */
+export default class CacheDriverPlugin implements Plugin {
+  get metadata(): PluginMetadata {
+    return {
+      name: 'cache_driver',
+      depends: ['cache', 'dom_events'],
+    };
+  }
+
+  /**
+   * A reference to the registry of plugins for internal use.
+   */
+  core: PluginRegistry;
+
+  /**
+   * Constructs a new instance of this plugin and binds the necessary
+   * callbacks.
+   */
+  constructor() {
+    this.handleRequest = this.handleRequest.bind(this);
+  }
+
+  /**
+   * Saves the plugin registry for later use. This plugin does not
+   * expose a value.
+   *
+   * @param plugins the plugin registry to use.
+   */
+  register(plugins: PluginRegistry): void {
+    this.core = plugins;
+  }
+
+  /**
+   * Registers a number of event listeners.
+   * The following events are listened for:
+   *
+   * - [[CACHE_REQUEST]]
```

Comment:
This feels unnecessary when we have only one item, but it's not a big deal.

Reply:
I'll leave it as-is for consistency and for ease of future modification.